### PR TITLE
Documentation for integrating Auth0 with the Swift SDK

### PR DESCRIPTION
**Pull request by turnipdabeets**

### CODE REVIEW

**Comment by Will5** [comment]
LGTM

**Review by Will5 — APPROVED** [review]

File: docs/sdks/swift-sdk/integrate-with-auth0.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+ - [Integrate With Auth0](/guides/sso-integrations/integrate-with-auth0)
+ - [Swift SDK Overview](overview)
+
+Before calling [`Embedded.shared.authenticate`](overview#authentication), we must [Authorize With Auth0](#authorize-with-auth0).
+
+### Authorize With Auth0
+
+ - Step 1: Configuring the Authenticator Config
+
+Make sure the [Authentication Config](/docs/v1/platform-overview/authenticator-config#embedded) in the Beyond Identity Console is set to type `Embedded` and that the Invoke URL points to your application with either an App Scheme or a Universal Link.
```
Comment (ismailmustafa) [inline docs/sdks/swift-sdk/integrate-with-auth0.md:19]: authentication -> authenticator

File: docs/sdks/swift-sdk/integrate-with-auth0.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+sidebar_position: 3
+---
+
+This guide describes how to configure Auth0 to delegate to Beyond Identity for authentication during an OAuth2 authorization flow. 
+
+### Prerequisites
+
+ - [Integrate With Auth0](/guides/sso-integrations/integrate-with-auth0)
+ - [Swift SDK Overview](overview)
+
```
Comment (ismailmustafa) [inline docs/sdks/swift-sdk/integrate-with-auth0.md:12]: Can you add the images that @Will5 created to the Auth0 and Okta guides?

File: docs/sdks/swift-sdk/integrate-with-auth0.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+To begin the authentication flow, start an `ASWebAuthenticationSession`, and load the OAuth2 authorization request URL provided by Okta. Make sure `prefersEphemeralWebBrowserSession` is set to `false` as the browser will need to share cookies or other browsing data between the authentication session.
+
+```javascript
+let session = ASWebAuthenticationSession(
+    url: viewModel.auth0URL,
+    callbackURLScheme: viewModel.callbackScheme
+    completionHandler: { (url, error) in }
+)
+session.prefersEphemeralWebBrowserSession = false
```
Comment (ismailmustafa) [inline docs/sdks/swift-sdk/integrate-with-auth0.md:31]: Should we mention the differences between setting this to true/false?

**Review by ismailmustafa — CHANGES_REQUESTED** [review]

File: docs/sdks/swift-sdk/integrate-with-auth0.md (hunk 3 above)
Comment (turnipdabeets) [reply docs/sdks/swift-sdk/integrate-with-auth0.md:31]: This is written above the example. Do you want to expand on it?

> Make sure `prefersEphemeralWebBrowserSession` is set to `false` as the browser will need to share cookies or other browsing data between the authentication session.

File: docs/sdks/swift-sdk/integrate-with-auth0.md (hunk 1 above)
Comment (turnipdabeets) [reply docs/sdks/swift-sdk/integrate-with-auth0.md:19]: Sure, I'll update this in all the places

File: docs/sdks/swift-sdk/integrate-with-auth0.md (hunk 3 above)
Comment (ismailmustafa) [reply docs/sdks/swift-sdk/integrate-with-auth0.md:31]: I would probably change "will need" to "may need". We don't required cookies but Okta/Auth0 may.

File: docs/sdks/swift-sdk/integrate-with-auth0.md (hunk 2 above)
Comment (turnipdabeets) [reply docs/sdks/swift-sdk/integrate-with-auth0.md:12]: Updated and added an SDK section like android

File: docs/sdks/swift-sdk/integrate-with-auth0.md (hunk 3 above)
Comment (turnipdabeets) [reply docs/sdks/swift-sdk/integrate-with-auth0.md:35]: It seems necessary in this case. Okta works in both cases but Auth0 doesn't work without setting this to false.

**Review by ismailmustafa — APPROVED** [review]

**Pull request merged**